app/controllers/v1/payments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class V1::PaymentsController < V1::BaseController
+
+  def create
+
+    charge = Stripe::Charge.create(payment_params)
+
+    person = Person.create_or_update(person_params)
+
+    person.create_action(action_params.symbolize_keys)
+
+    render json: { status: 'success' }
+
+  rescue Stripe::CardError => e
+    render json: { error: e.message }
+  end
+
+  private
+
+  def action_params
+    amount = payment_params['amount'].to_f/100.0
```

Comment:
oh yeah, i didn't realizie it was a float.  It should be a int  (donation_amt_in_pennies).